src/v3/src/components/WidgetMessageContainer/WidgetMessageContainer.tsx

```diff
@@ -11,14 +11,57 @@
  */
 
 import { List, ListItem } from '@mui/material';
-import { Box, Typography } from '@okta/odyssey-react-mui';
+import { Box, Link, Typography } from '@okta/odyssey-react-mui';
+import { HTMLReactParserOptions } from 'html-react-parser';
 import { FunctionComponent, h } from 'preact';
 import React from 'preact/compat';
 
-import { WidgetMessage } from '../../types';
+import { useHtmlContentParser } from '../../hooks';
+import { ListStyleType, WidgetMessage, WidgetMessageLink } from '../../types';
 
-const WidgetMessageContainer: FunctionComponent<{ message?: WidgetMessage }> = (props) => {
-  const { message } = props;
+const WidgetMessageContainer: FunctionComponent<{
+  message?: WidgetMessage,
+  parserOptions?: HTMLReactParserOptions,
+  linkVariant?: 'monochrome' | 'body1',
+}> = (props) => {
+  const { message, parserOptions, linkVariant } = props;
+
+  const renderTitle = (title?: string) => (title ? (
+    <Typography
+      component="h2"
+      variant="h6"
+    >
+      {title}
+    </Typography>
+  ) : null);
+
+  const renderLinks = (links?: WidgetMessageLink[], listStyleType?: ListStyleType) => (links ? (
+    <List
+      className="custom-links"
+      disablePadding
+      dense
+      sx={{ pl: 4, listStyleType: listStyleType ?? 'disc' }}
+    >
+      {links.map((link) => (
+        <ListItem
+          sx={{
+            paddingLeft: 0,
+            display: 'list-item',
+          }}
+          key={link.url}
+        >
```

Comment:
ah yes youre right good catch this would affect the custom access denied error terminal view.  That view is not supposed to have a list style and padding left.  I will change it so it does not default to `disc` style type, and make the padding left conditional.